core/extensions/helpers/css.php

```diff
@@ -53,24 +53,24 @@ class Css
      * @param string $file nombre del archivo a añadir
      * @param array $dependencies  archivos que son requisito del archivo a añadir
      */
-    public static function add( $file, array $dependencies = [] )
+    public static function add($file, array $dependenciess = [])
     {
-        self::$_css[$file] = $file;
-        foreach ($dependencies  as $file) self::$_dependencies [$file] = $file;
+        self::$css[$file] = $file;
+        foreach ($dependenciess as $file) self::$dependencies[$file] = $file;
```

Comment:
__[PHP_CodeSniffer]__

Inline control structures are not allowed


<!-- Comment by [SideCI](https://sideci.com) -->